cmd/terminal.go

```diff
@@ -45,22 +45,20 @@ func TerminalGenerate(out io.Writer, cmd *cobra.Command, args []string) error {
 	var PS1 = `
 __odo_ps1() {
 
-    # Get current context
-    COMPONENT_GET=$(odo component get --skip-connection-check | sed "s/The current component is: //g")
-		if [ $? != 0 ]; then
-		COMPONENT="Unable to retrieve component"
-		else
-    COMPONENT=$(echo $COMPONENT_GET | sed "s/The current component is: //g")
-		fi
-
 		# Get application context
-    APP_GET=$(odo application get --skip-connection-check | sed "s/The current application is: //g")
-		if [ $? != 0 ]; then
-		APP="Unable to retrieve app"
-		else
-    APP=$(echo $APP_GET | sed "s/The current application is: //g")
+    APP=$(odo application get -q --skip-connection-check)
```

Comment:
Is the `--skip-connection-check` required? Even if the cluster is down, it's showing me the app/component name

Reply:
Yes, it's required. If a connection check is used, each call will take 1+ more seconds.

Reply:
But if the cluster is down, it shows me the last active component/app name

Reply:
@mik-dass That's intentional